docs/installationguide.md

```diff
@@ -256,6 +256,7 @@ The ones relating specific Ultralight 2.0 bindings are described in the followin
 | IOTA_MQTT_AVOID_LEADING_SLASH | mqtt.avoidLeadingSlash  |
 | IOTA_MQTT_CLEAN               | mqtt.clean              |
 | IOTA_MQTT_CLIENT_ID           | mqtt.clientId           |
+| IOTA_MQTT_DISABLED            | mqtt.disabled           |
```

Comment:
config.js should be modified to include the new mqtt.disabled and amqp.disables settings (in commented way). Note config.js is used also somehow to document configurations.

Reply:
Fixed bebba29